src/components/view/appView.ts

```diff
@@ -0,0 +1,28 @@
+import { ResponceModelAll, ResponceModelSources } from '../models/index.model';
+import News from './news/news';
+import Sources from './sources/sources';
+
+export class AppView {
+  private news: News;
+  private sources: Sources;
+  constructor() {
+    this.news = new News();
+    this.sources = new Sources();
+  }
+
+  drawNews(data: ResponceModelSources | ResponceModelAll) {
+    if ('articles' in data) {
```

Comment:
don't we know what type of data is received by drawNews? do we really need type narrowing?